src/GitHub.InlineReviews/SampleData/DiffCommentThreadViewModelDesigner.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System;
+using System.Diagnostics.CodeAnalysis;
 using GitHub.InlineReviews.ViewModels;
 
 namespace GitHub.InlineReviews.SampleData
 {
+    [SuppressMessage("Microsoft.Performance", "CA1812:AvoidUninstantiatedInternalClasses")]
     class DiffCommentThreadViewModelDesigner : IDiffCommentThreadViewModel
     {
         public string DiffHunk { get; set; }
         public int LineNumber { get; set; }
+        public event EventHandler Disposed = delegate { };
```

Comment:
This member can just be removed: it's not implementing an interface or anything.